src/libs/scene_utils/cesium/bufferToJson.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
FYI we'll need to add the license header to each file

Reply:
I'll add those lines and remove the separate LICENSE file